style.css

```diff
@@ -0,0 +1,68 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background-color: #151515;
+}
+
+body {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    color: #d3d3d3;
+}
+
+/**
+// Header Styles
+**/
+
+header h1 {
+    font-size: 1.2em;
+    color: #d3d3d3;
+    top: 5;
+    left: 0;
+    padding-bottom: 0.5em;
+    padding-top: 0.5em
+    /* position: absolute; */
+}
+
+input.searchbar {
```

Comment:
A background color could be added to the area above the navbar so that the navbar and the area above it looks the same. Possibly even no background at all or just a bottom border to separate it from the content below as other options.